lib/experimental/Navigation/Sidebar/CompanySelector/index.tsx

```diff
@@ -83,14 +103,32 @@ const Selector = ({
           "aria-label": `${company.name} logo`,
         } satisfies AvatarVariant,
       })),
-    [companies]
+      ...(additionalOptions ? ["separator" as const] : []),
+      ...(additionalOptions?.map((option) => ({
```

Comment:
can't we spread the whole `additionalOptions` object as is? 
```
...additionalOptions
```

Reply:
Right! That's a lot simpler, thanks 🙏